doc/user-guide/io.rst

```diff
@@ -19,6 +19,80 @@ format (recommended).
 
     np.random.seed(123456)
 
+You can `read different types of files <https://docs.xarray.dev/en/stable/user-guide/io.html>`_
+in `xr.open_dataset` by specifying the engine to be used:
+
+.. ipython:: python
+
+    import xarray as xr
+
+    xr.open_dataset("my_file.grib", engine="cfgrib")
```

Comment:
@max-sixty This is actually what's causing the docs build to fail. I copy-pasted it from the Xarray-tutorial, so I think it's just supposed to be rendered (not run). Do you know the directive offhand to stop it executing?

Reply:
Ah yes that explains it! This is being run at the moment... Should we just remove it? Or there's likely a way to have it not run if we only want to render...